unit-tests/live/d500/test-hw-timestamp.py

```diff
@@ -0,0 +1,57 @@
+# License: Apache 2.0. See LICENSE file in root directory.
+# Copyright(c) 2024 Intel Corporation. All Rights Reserved.
+
+# test:donotrun
+
+import pyrealsense2 as rs
+from rspy import test
+import time
+
+
+dev = test.find_first_device_or_exit()
+depth_sensor = dev.first_depth_sensor()
+is_global_time_enabled_orig = False
+fps = 30
+
+
+def check_hw_ts_right_before_sensor_ts(frame):
+    frame_ts_supported =  frame.supports_frame_metadata(rs.frame_metadata_value.frame_timestamp)
+    sensor_ts_supported = frame.supports_frame_metadata(rs.frame_metadata_value.sensor_timestamp)
+    test.check(frame_ts_supported and sensor_ts_supported)
+    hw_ts = frame.get_frame_metadata(rs.frame_metadata_value.frame_timestamp)
+    sensor_ts = frame.get_frame_metadata(rs.frame_metadata_value.sensor_timestamp)
+    delta = sensor_ts - hw_ts
+    test.check(delta < (1/fps * 10^6))
```

Comment:
Should also check `delta > 0`.
If hw_ts is after sensor_ts you will get a negative delta

Reply:
Done